src/main/jbake/content/blog/2017/01/13/events-aggregation-extension.adoc

```diff
@@ -0,0 +1,189 @@
+= Extending Complex Event Processing in Hawkular Alerting
+Lucas Ponce
+2017-01-13
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, alerts, complex, events, processing, stream, polling
+
+== Stream Processing versus Polling Processing
+
+Hawkular Alerting uses different techniques to detect behaviours by defining rules.
+
+The main implementation is based in a Stream Processing design.
+Hawkular Alerting analyzes streams of data and events ordered in a timeline searching for matches against conditions.
+This technique is also known as Event Processing (or Complex Event Processing) specially when the processing of incoming data might need multiple conditions.
```

Comment:
Regardless of the CEP definition, if the description is not changing it needs a couple of small grammatical changes:

'and events ordered in a timeline searching for matches against conditions.'  --> 'and events, ordered in a timeline, searching for condition matches.'

'specially'  -> especially (or just omit 'specially' completely)